spec/integration/message_queue_publishing_spec.rb

```diff
@@ -3,6 +3,10 @@
 RSpec.describe "Message queue publishing" do
   include RandomContentHelpers
 
+  before do
+    Sidekiq::Worker.clear_all
+  end
```

Comment:
Do you know why we need this? It seems peculiar as [Sidekiq jobs are configured to run inline](https://github.com/alphagov/publishing-api/blob/d1c4d517c3d1b1dc85d1d168fbe04321c3144fcc/spec/support/sidekiq.rb#L2) and so I wondered where the state that caused the need for this came from.

Reply:
Thanks for catching, this wasn't necessary - I think I added this in response to a failing test which I haven't been able to reproduce. I've removed it.